packages/types/config/features.ts

```diff
@@ -0,0 +1,16 @@
+export interface NuxtConfigurationFeatures {
+  store?: boolean
+  layouts?: boolean
+  meta?: boolean
+  middleware?: boolean
+  transitions?: boolean
+  deprecations?: boolean
+  validate?: boolean
+  asyncData?: boolean
+  fetch?: boolean
+  clientOnline?: boolean
+  clientPrefetch?: boolean
+  clientUseUrl?: boolean
+  componentAliases?: boolean
+  componentClientOnly?: boolean
+}
```

Comment:
Should be `features.d.ts` over `features.ts`, and could you please sort alphabetically ? Thanks :)